test/testenv/appframework_utils.go

```diff
@@ -75,6 +75,9 @@ var AppLocationV1 = "appframework/v1apps/"
 // AppLocationV2 Location of apps on S3 for V2 Apps
 var AppLocationV2 = "appframework/v2apps/"
 
+// AppLocationDisabledApps Location of apps on S3 for Disabled Apps
+var AppLocationDisabledApps = "appframework/Disabledapps/"
```

Comment:
Create a follow up task to modify upload apps on demand. Rather than having pre-disabled apps. That way we will have less artifacts to maintain.